ext/spl/spl_directory.stub.php

```diff
@@ -150,6 +150,67 @@ public function __toString(): string {}
 
 class FilesystemIterator extends DirectoryIterator
 {
+    /**
+     * @var int
+     * @cvalue SPL_FILE_DIR_CURRENT_MODE_MASK
+     */
+    public const CURRENT_MODE_MASK = UNKNOWN;
+    /**
+     * @var int
+     * @cvalue SPL_FILE_DIR_CURRENT_AS_PATHNAME
+     */
+    public const CURRENT_AS_PATHNAME = UNKNOWN;
+    /**
+     * @var int
+     * @cvalue SPL_FILE_DIR_CURRENT_AS_FILEINFO
+     */
+    public const CURRENT_AS_FILEINFO = UNKNOWN;
+    /**
+     * @var int
+     * @cvalue SPL_FILE_DIR_CURRENT_AS_SELF
+     */
+    public const CURRENT_AS_SELF = UNKNOWN;
+    /**
+     * @var int
+     * @cvalue SPL_FILE_DIR_KEY_MODE_MASK
+     */
+    public const KEY_MODE_MASK = UNKNOWN;
+    /**
+     * @var int
+     * @cvalue SPL_FILE_DIR_KEY_AS_PATHNAME
+     */
+    public const KEY_AS_PATHNAME = UNKNOWN;
+    /**
+     * @var int
+     * @cvalue SPL_FILE_DIR_FOLLOW_SYMLINKS
+     */
+    public const FOLLOW_SYMLINKS = UNKNOWN;
+    /**
+     * @var int
+     * @cvalue SPL_FILE_DIR_KEY_AS_FILENAME
+     */
+    public const KEY_AS_FILENAME = UNKNOWN;
+    /**
+     * @var int
+     * @cvalue SPL_FILE_NEW_CURRENT_AND_KEY
+     */
+    public const NEW_CURRENT_AND_KEY = UNKNOWN;
```

Comment:
```suggestion
    public const NEW_CURRENT_AND_KEY = self::CURRENT_AS_FILEINFO|self::KEY_AS_FILENAME;
```
Would be the idea

Reply:
Hmmm, great idea, I'll try it if gen_stub properly support it

Reply:
No, unfortunately, constant evaluation doesn't work properly :( So I'll stay with my original implementation